build/run

```diff
@@ -120,14 +130,19 @@ if [ -z "${DISABLE_NESTED_DOCKER}" ]; then
     -v ${HOME}/.docker/config.json:${BUILDER_HOME}/.docker/config.json \
     -v /var/run/docker.sock:/var/run/docker.sock"
 fi
+DOCKER_API=''
+
+if (( $(bc <<< "$DOCKER_SERVER_API < $DOCKER_CLIENT_API") )); then
+    DOCKER_API="-e DOCKER_API_VERSION=${DOCKER_SERVER_API}"
+fi
 
 docker run \
     --rm \
     -e GOPATH="${BUILDER_HOME}/go" \
     -e WORKDIR="${BUILDER_HOME}/go/.work" \
     -e GO_PKG_DIR="" \
     -e GITHUB_TOKEN \
-    -e VERSION \
+    -e VERSION $DOCKER_API \
```

Comment:
Concerning the double commit. Got bitten by git. Still stuck with svn 99% of my time. Hope to get better quickly ;-)